model_analyzer/record/types/gpu_utilization.py

```diff
@@ -40,11 +40,23 @@ def __init__(self, value, device=None, timestamp=0):
         super().__init__(value, device, timestamp)
 
     @staticmethod
-    def header(aggregation_tag=None):
+    def aggregation_function():
```

Comment:
Same, and why is the function static?  Same about the header function.

Reply:
Because I think the header and aggregation function belong to the class rather than the individual instance of the metric, but perhaps the aggregation function would need to be changed within a single run of model_analyzer ? It will be complicated to passs in the aggregation function in the constructor because a lot of the time we are not explicit about which metric we are constructing.

Reply:
Ok, I see. Probably because we don't change the aggregation function it should be fine.